hudi-common/src/test/java/org/apache/hudi/common/testutils/HoodieTestTable.java

```diff
@@ -143,6 +143,10 @@ public static String makeNewCommitTime(int sequence) {
     return String.format("%09d", sequence);
   }
 
+  public static String makeNewCommitTimeInHudiFormat(int sequence) {
```

Comment:
Love to not add more methods here. can we get `makeNewCommitTime` work as-is? cc @xushiyan

Reply:
In any case : rename `makeNewCommitTime(int ..)` ?

Reply:
> can we get makeNewCommitTime work as-is?

`makeNewCommitTime` actually creates the instants with a size of 9 digits while the actual instants are created with a size of 14 digits. I was facing issues when comparing the timestamps in test cases due to the difference of sizes, hence had to introduce a new method. I wanted to understand why did we originally have `makeNewCommitTime` create instants of size 9 digits? @vinothchandar

Reply:
> In any case : rename makeNewCommitTime(int ..) ?

Let me rework this and pass the size of instant as a parameter in the existing method itself.